src/python/WMComponent/RucioInjector/RucioInjectorPoller.py

```diff
@@ -474,6 +475,8 @@ def insertContainerRules(self):
             else:
                 # then it's a T0 container placement
                 ruleKwargs['priority'] = 4
+                if not rseName.endswith("_Tape") and lifetime > 0:
```

Comment:
Hello @amaltaro , Alan, About this line `lifetime >0`, how mandatory this PR needs to include that additional statement ?. This due that if in the future from T0 Configfile, T0 decides to add to a container the value of `None` this will give an exception.

Reply:
The database column is defined as an integer. If we are retrieving this data from the database with a `None` value, then I would say we overlooked something and it needs to be fixed. Right?

Reply:
Hi Alan, I double-check with my notes and the requirements for this development and the default value will be 18 months and all the datasets shipping to disk sites must have this new attribute of lifetime. So, everything is fine with this PR.